variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "prefix" {
+description = "The prefix used for all resources in this environment"
+}
+
+variable "location" {
+description = "The Azure location where all resources in this deployment should be created"
+default = "uksouth"
+}
+
+variable "GITHUBID" {
+description = "The githubid"
+default = "47190936"
+}
+
+variable "WEBAPPLICATIONCLIENT" {
+description = "client web"
+default = "88de9a9a5377861febef"
+}
+variable "CONNECTIONSTRING" {
+description = "mongo connection"
+default = "mongodb://tododatabaseproject:JAyasz79f7yRR2zqi34xJG5PkjXbaRlvKJC4XlUYrqezapD97bHUF9WFeIpoDxXdaUNhby7ohYnSYK5Pommsfw==@tododatabaseproject.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@tododatabaseproject@"
```

Comment:
Here you have several secrets that have been committed to your version control in plain text! I'll comment more on this in your M12 review, but we definitely want to rotate all of these